app/services/trading_strategies.py

```
@@ -4215,6 +4655,459 @@ async def _estimate_daily_volatility(self, symbol: str) -> float:
+                return 0.01   # 1%
+        except Exception:
+            return 0.025
+
+    async def _get_available_options(self, symbol: str) -> List[Dict]:
+        """Get available options for a symbol."""
+        try:
+            # Get real market price instead of hardcoded values
+            price_data = await self._get_symbol_price("auto", symbol)
+            if not price_data or not price_data.get("success"):
+                # Fall back with clear error message
+                raise ValueError(f"Unable to fetch price for {symbol}")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)